stock_production_lot_responsible/i18n/es.po

```diff
@@ -0,0 +1,65 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* stock_production_lot_responsible
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-04-27 09:08+0000\n"
+"PO-Revision-Date: 2021-04-27 09:08+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: stock_production_lot_responsible
+#: model:ir.model.fields,field_description:stock_production_lot_responsible.field_stock_production_lot__display_name
+#: model:ir.model.fields,field_description:stock_production_lot_responsible.field_stock_quant__display_name
+msgid "Display Name"
+msgstr "Nombre mostrado"
+
+#. module: stock_production_lot_responsible
+#: model:ir.model.fields,field_description:stock_production_lot_responsible.field_stock_production_lot__id
+#: model:ir.model.fields,field_description:stock_production_lot_responsible.field_stock_quant__id
+msgid "ID"
+msgstr ""
+
+#. module: stock_production_lot_responsible
+#: model:ir.model.fields,field_description:stock_production_lot_responsible.field_stock_quant__ref
+#: model_terms:ir.ui.view,arch_db:stock_production_lot_responsible.quant_search_view
+msgid "Internal Reference"
+msgstr "Referencia interna"
+
+#. module: stock_production_lot_responsible
+#: model:ir.model.fields,help:stock_production_lot_responsible.field_stock_quant__ref
+msgid ""
+"Internal reference number in case it differs from the manufacturer's "
+"lot/serial number"
+msgstr "Lote/Nº de serie"
```

Comment:
A ver.... El mensaje en Ingles es 
```
Internal reference number in case it differs from the manufacturer's lot/serial number
```

Por lo que la traduccion es:
```
Número de referencia interno en caso de que difiera del número de Lote/Serie del fabricante
```

Lo que hace que el *código* sea:
```
msgstr ""
"Número de referencia interno en caso de que difiera del número de Lote/Serie"
" del fabricante"
```